chunky/src/java/se/llbit/chunky/model/ChainModel.java

```diff
@@ -33,11 +33,22 @@
           },
           Math.toRadians(45));
 
-  public static boolean intersect(Ray ray) {
+  private static Quad[][] axis = new Quad[3][];
+
+  static {
+    Quad[] alignX = Model.rotateZ(alignY);
+    Quad[] alignZ = Model.rotateX(alignY);
+
+    axis[0] = alignY;
+    axis[1] = alignX;
+    axis[2] = alignZ;
+  }
+
+  public static boolean intersect(Ray ray, int data) {
```

Comment:
Should be `String axis` instead of `int data` so that the array index number thing is contained in the model class.